x-pack/test/functional/apps/ml/anomaly_detection/saved_search_job.ts

```diff
@@ -146,7 +146,7 @@ export default function ({ getService }: FtrProviderContext) {
           bucket_count: '479',
           earliest_record_timestamp: '2016-02-07 00:00:00',
           latest_record_timestamp: '2016-02-11 23:59:54',
-          input_record_count: '34,415',
+          input_record_count: '86,274',
```

Comment:
Not sure if this change was intended. But it's failing on my local deployment and on CI. I think we should revert this.

Reply:
Reverted here https://github.com/elastic/kibana/pull/85726/commits/958b660e3b151b77001a6dc7bb460b5ae278a84f